esctest/escutil.py

```diff
@@ -1,26 +1,32 @@
+import functools
+import traceback
+
 import esc
 import escargs
 import esccmd
 import escio
-from esclog import LogDebug, LogInfo, LogError, Print
+from esclog import LogInfo, LogDebug
 import esctypes
 from esctypes import Point, Size, Rect
-import functools
-import traceback
 
 gNextId = 1
 gHaveAsserted = False
 
+CHAR_SIZE_PIXELS = Size(0, 0)
+FRAME_SIZE_PIXELS = Size(0, 0)
+SCREEN_SIZE_PIXELS = Size(0, 0)
+CAN_QUERY_SHELL_SIZE = -1
```

Comment:
I've been using SCREAMING_SNAKE_CASE for constants and gCamelCase for global variables. These are more like global variables than constants since they get assigned to, so please name them like `gCharSizePixels`, etc.

Reply:
ok